packages/components/pie-modal/src/modal.scss

```diff
@@ -22,6 +22,18 @@
     --modal-bg-color: var(--dt-color-container-default);
     --modal-elevation: var(--dt-elevation-04);
 
+    &:focus-visible {
```

Comment:
I didn't notice this the first time around, but the ordering of definitions here makes it pretty hard to scan (imo).

I'm pretty relaxed on ordering for the most part, but I do think we should keep to the structure of:

- Custom props defined at the start of a block
- Any block-specific style declarations
- Media queries for that block
- and then any other descendent or pseudo-selectors after that

Could you put in a PR to move this around based on that?